langchain4j/src/test/java/dev/langchain4j/service/AiServicesWithToolsIT.java

```diff
@@ -48,13 +54,6 @@ static Stream<ChatLanguageModel> models() {
                         .temperature(0.0)
                         .logRequests(true)
                         .logResponses(true)
-                        .build(),
-                MistralAiChatModel.builder()
```

Comment:
Is there a reason for removing this model?

Reply:
I dint had Mistral Locally, so removed. we can add it back.

Reply:
That is cloud mistral.

But yes, please bring it back

Reply:
Its already there. Some how it doesn't show here. Please merge and check in the main branch